### ISSUE TITLE
Better output style with Termwind

### ISSUE THREAD

**Issue by roberto-butti** [Hi-Folks/lara-lens#70]
Adding Termwind we could achieve a better output.

The suggestion comes from a Twitter thread : https://twitter.com/jbrooksuk/status/1540241600713998336 

We should add Termwind package: https://github.com/nunomaduro/termwind

For showing list of Label / Value we could use this snippet (or at least we coul start from this one ,and try to improve that):

```php
<?php
require "vendor/autoload.php";

use function Termwind\{render,terminal};

$separator= str_pad("", terminal()->width(),".");
$label = "Title for the line";
$value = "Well Done";
render(<<<HTML
    <div class="mx-1 my-1">
        <div class="flex space-x-1">
            <span class="text-gray-300">{$label}</span>
            <span class="text-gray-500 flex-1">{$separator}</span>
            <span class="text-white font-bold">{$value}</span>
        </div>
    </div>
HTML);
```

**Comment by roberto-butti** [comment]
Some more examples : 
- https://github.com/laravel/pint/blob/main/resources/views/summary.blade.php
- https://twitter.com/RmeetsH/status/1543837138722111489

**Comment by roberto-butti** [comment]
Closed with feature #72 .
Thanks to @exodusanto
